parser/testdata/workflows/greetings-constants-file.sw.yaml

```diff
@@ -19,7 +19,7 @@ description: Greet Someone
 specVersion: "0.8"
 start:
   stateName: Greet
-constants: "testdata/constantsDogs.json"
+constants: "file://testdata/constantsDogs.json"
```

Comment:
It should work without the schema (since the default is `file://`). Do we have a test to verify it?

Reply:
Yes. Have the tests: `file://path/`, `file://../path/`, `./path/`, `/path/`